app/views/users/registrations/edit.html.erb

```diff
@@ -28,11 +28,11 @@ prepare_to_initialize("searchTextField");
 
   <div class="field">
     <%= f.label :address %><br />
-    <input id="searchTextField" type="text" size="50" placeholder="Enter a location" autocomplete="on" runat="server" value="<%= current_user.location_name %>" />
+    <input id="searchTextField" class="search" type="text" size="50" placeholder="Enter a location" autocomplete="on" runat="server" value="<%= current_user.location_name %>" />
 
-    <%= f.text_field :location_name, autofocus: true, id: 'addName', hidden: true %>
-    <%= f.text_field :location_lat, autofocus: true, id: 'addLat', hidden: true %>
-    <%= f.text_field :location_lng, autofocus: true, id: 'addLng', hidden: true %>
+    <%= f.text_field :location_name, autofocus: true, id: 'addName', class: 'add_name', hidden: true %>
```

Comment:
Instead of `add_name` class, I think something like `location_name` name will be better, and so on for lat, and lang